.travis.yml

```diff
@@ -1,38 +1,37 @@
 language: ruby
-sudo: false
 cache: bundler
 
 rvm:
-  - 2.5.3
-  - 2.4.5
-  - 2.3.8
-  - 2.2.9
-  - 2.1.10
+  - 2.7
+  - 2.6
+  - 2.5
+  - 2.4
+  - 2.3
+  - 2.2
+  - 2.1
   - ruby-head
+  - jruby-9.2.11.1
+  - jruby-head
 
 matrix:
   include:
-    - rvm: 2.5.3
+    - rvm: 2.5
       name: "Run Danger and Code Climate"
       before_script:
         - bundle exec danger
       after_script:
         - bundle exec codeclimate-test-reporter
-    - rvm: 2.5.3
+    - rvm: 2.5
       name: "Integration Tests"
       install:
         - for dir in spec/integration/*; do BUNDLE_GEMFILE=$dir/Gemfile bundle; done
       script:
         - set -e ; for dir in spec/integration/*; do BUNDLE_GEMFILE=$dir/Gemfile bundle exec rspec $dir; done
-    - rvm: 2.6.6
-      dist: bionic
     - rvm: rbx-3
       dist: trusty
       bundler_args: --retry 0
     - rvm: jruby-9.0.5.0
       dist: trusty
```

Comment:
Not directly part of this PR, but do you wish to continue testing Rubinius? It seems that for testing Rubinius, the only option is to use an EOL Ubuntu.

https://github.com/rubinius/rubinius/issues/3819
https://docs.travis-ci.com/user/languages/ruby/#rubinius

Reply:
Whoops, wrong line, should be on the `rbx-3` line, not `jruby`. My bad!

But yeah, do you want to keep testing rbx-3? It seems to fail anyway. We could switch it for testing with TruffleRuby instead, WDYT?

Also I'll try testing jruby 9.0 on xenial.

Reply:
Don't know what we want for Rubinius TBH. I was OK removing it a long time ago.